applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -8,13 +8,103 @@
 #define _LE_AUDIO_H_
 
 #include <zephyr.h>
+#include <bluetooth/audio/audio.h>
 
 #define DEVICE_NAME_PEER CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
 
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
 #define LE_AUDIO_PRES_DELAY_US 10000u
 
+#if CONFIG_TRANSPORT_CIS
+#define BT_AUDIO_LC3_UNICAST_PRESET_RECOMMENDED(_loc, _stream_context)                             \
+	BT_AUDIO_LC3_PRESET(                                                                       \
+		BT_CODEC_LC3_CONFIG(                                                               \
+			BT_CODEC_CONFIG_LC3_FREQ_48KHZ, BT_CODEC_CONFIG_LC3_DURATION_10, _loc,     \
+			LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 1, _stream_context),         \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 2u,     \
+					     20u, LE_AUDIO_PRES_DELAY_US))
+
+#if CONFIG_AUDIO_SOURCE_USB
+/* Only 48kHz is supported when using USB */
+#if !CONFIG_BT_AUDIO_UNICAST_RECOMMENDED
+#error USB only supports 48kHz
```

Comment:
Bit strange to mix #error and BUILD_ASSERT

Reply:
Agee, will change to using #error both places

Reply:
OK, I still see some BUILD_ASSERTS in this PR though.

Reply:
Do you mean the BUILD_ASSERT in le_audio_bis_gateway.c that was not changed in this PR? Other than that I do not see any other BUILD_ASSERTS